subprojects/ide/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r50/ToolingApiIdeaModelCrossVersionSpec.groovy

```diff
@@ -22,13 +22,17 @@ import org.gradle.integtests.tooling.fixture.ToolingApiVersion
 import org.gradle.tooling.model.idea.IdeaContentRoot
 import org.gradle.tooling.model.idea.IdeaModule
 import org.gradle.tooling.model.idea.IdeaProject
+import org.gradle.util.ToBeImplemented
+import org.junit.Ignore
 
 /**
  * NOTE: Starting with Gradle 5.0 the contract of IdeaModule#sourceDirs and IdeaModule#testSourceDirs changes in
  * a way that the resource directories are excluded.
  */
 @ToolingApiVersion(">=5.0")
 @TargetGradleVersion(">=5.0")
+@Ignore
+@ToBeImplemented
```

Comment:
Do we have an issue for that?

Reply:
https://github.com/gradle/gradle/issues/6312

Reply:
Should we reference it here, e.g. using `@Issue`?